### PR TITLE
Update lumina-desktop_sv.ts

### PR DESCRIPTION
**Pull request by bittin**
Updated Swedish translation

### CODE REVIEW

File: src-qt5/core/lumina-desktop/i18n/lumina-desktop_sv.ts [hunk 1]
```diff
@@ -40,12 +40,12 @@
     <message>
         <location filename="../desktop-plugins/applauncher/AppLauncherPlugin.cpp" line="74"/>
         <source>Launch %1</source>
-        <translation type="unfinished">Starta %1</translation>
+        <translation>Starta %1">Starta %1</translation>
```
Comment (beanpole135) [inline src-qt5/core/lumina-desktop/i18n/lumina-desktop_sv.ts:43]:
```suggestion
        <translation>Starta %1</translation>
```

File: src-qt5/core/lumina-desktop/i18n/lumina-desktop_sv.ts [hunk 2]
```diff
@@ -827,15 +827,15 @@
         <location filename="../LDesktop.cpp" line="662"/>
         <location filename="../LDesktop.cpp" line="683"/>
         <source>Desktop</source>
-        <translation type="unfinished">Skrivbord</translation>
+        <translation>Skrivbord">Skrivbord</translation>
```
Comment (beanpole135) [inline src-qt5/core/lumina-desktop/i18n/lumina-desktop_sv.ts:830]:
```suggestion
        <translation>Skrivbord</translation>
```

**Review by beanpole135 — CHANGES_REQUESTED** [review]
A ton of these have duplication translation strings with half-tags.
I suggested fixes for a couple of them, but you will need to go through and fix the others the same way.

**Comment by q5sys** [comment]
This work has been redone with later PRs

**Pull request closed without merging**